emissionscheck_alb/baseline_model.py

```diff
@@ -0,0 +1,59 @@
+import pandas as pd
+import numpy as np
+
+
+def run_baseline_model(features_train, features_test, target_train, target_test, model, random_state=None, class_weight=None, max_fpr=None):
+    """
+    Run logistic regression or SVC  
+    
+    """
+    from sklearn.pipeline import make_pipeline
+    from sklearn.preprocessing import MinMaxScaler
+    from sklearn.metrics import roc_auc_score, plot_confusion_matrix, classification_report
+    from sklearn.model_selection import cross_val_score
+
+    if model == "logreg":
+
+        from sklearn.linear_model import LogisticRegression
+        lr = LogisticRegression(class_weight=class_weight, random_state=random_state)
+
+        clf_pipeline = make_pipeline(MinMaxScaler(), lr)
+
+    elif model == "svc":
+
+
+        from sklearn.svm import SVC
+        svc = SVC(class_weight=class_weight)
+
+        clf_pipeline = make_pipeline(MinMaxScaler(), svc)
```

Comment:
i think it's really cool how you are modularizing the use of the models. i was thinking we could have a class, and when we initialize it, we pass the name of the model and variables we want to use.

Reply:
I'm also open to that! I don't exactly know how that works so if you have time/are interested feel free to refactor this script to do that. Or maybe implement a script like this for future models.